wikibase/queryService/api/Sparql.js

```diff
@@ -213,12 +214,7 @@ wikibase.queryService.api.Sparql = ( function ( $ ) {
 			debug: request.responseText
 		};
 
-		if ( request.status === 0 || exception ) {
-			error.code = ERROR_CODES.SERVER;
-			error.message = exception.message;
-		}
-
-		try { // extract error from server response
+		try {//extract error from server response
```

Comment:
I think the if ( request.status === 0 || exception ) is indicating client side errors, these should stay at this location